sklearn/manifold/tests/test_t_sne.py

```diff
@@ -261,6 +263,8 @@ def test_preserve_trustworthiness_approximately(method, init):
     assert t > 0.85
 
 
+# TODO: Remove ignore_warnings in 1.2 when default init changes to 'pca'
+@ignore_warnings(category=FutureWarning)
```

Comment:
Can we update the `ignore_warnings` here to use `pytest.mark.filterwarnings` so we are explicit about which warnings we are catching?

```suggestion
@pytest.mark.filterwarnings("ignore:.*TSNE will change.*:FutureWarning")
```